scripts/surgical_breakdown_executor.py

```
@@ -0,0 +1,272 @@
+                  estimated_lines: int, phase: int) -> bool:
+        """Create a new micro-PR."""
+        branch_name = f"feat/dl-pr-{pr_number:02d}"
+        
+        print(f"🏗️  Creating PR-{pr_number}: {description}")
+        print(f"   Branch: {branch_name}")
+        print(f"   Files: {len(files)}")
+        print(f"   Estimated lines: {estimated_lines}")
+        
+        try:
+            # Create branch
+            subprocess.run(["git", "checkout", "-b", branch_name], check=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)